<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inforOut</class>
 <widget class="QDialog" name="inforOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="w_Position"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>纬度</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>经度</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>城市</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>可达城市数量</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="j_Position"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>人口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="cityName"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="cityConnect"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="population"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
